gravitee-am-ui/src/app/domain/applications/application/advanced/password-policy/password-policy.component.ts

```diff
@@ -112,6 +112,13 @@ export class PasswordPolicyComponent implements OnInit {
   }
 
   update() {
+    if (this.passwordHistoryEnabled) {
```

Comment:
Sonar might moan and say this outer `if` can be merged with the inner (i.e. give it four conditions). Suggest something like

`
      if (this.passwordHistoryEnabled && (!this.oldPasswords || (this.oldPasswords < 1 || this.oldPasswords > 24))) {
        this.snackbarService.open("Number of old passwords must be within the range [1, 24]")
        return;
      }
    `